build_files/bluefin-changes.sh

```diff
@@ -31,7 +31,7 @@ if [[ "${BASE_IMAGE_NAME}" = "silverblue" ]]; then
     find /usr/share/glib-2.0/schemas/ -type f ! -name "*.gschema.override" -exec cp {} /tmp/bluefin-schema-test/ \;
     cp /usr/share/glib-2.0/schemas/zz0-bluefin-modifications.gschema.override /tmp/bluefin-schema-test/
     echo "Running error test for bluefin gschema override. Aborting if failed."
-    glib-compile-schemas --strict /tmp/bluefin-schema-test
+    glib-compile-schemas /tmp/bluefin-schema-test
```

Comment:
what is the difference with having --strict removed?

Reply:
--strict is causing issues in GNOME <47 since the accent color dconf key doesn't exist in these lower versions.  The flag causes any errors in the dconf to fail, whereas with it excluded, those problematic keys are just excluded from the compiled data.

Reply:
If you omit `--strict` from here, you destroy the purpose of running this test.

But it's a good point that this gschema test should handle lower Gnome versions better.

So maybe this can be temporary removed than put back when this logic gets revamped to support gschema checking for both `:gts` & `:latest`? Would be good to comment that here.